src/Storage/AuthorizationCodeStorage.php

```diff
@@ -24,14 +24,16 @@ private function getUserIdByCode( $code ) {
 			return null;
 		}
 
+		$key = self::META_KEY_PREFIX . '_client_id_' . $code;
```

Comment:
I'm confused about this. Shouldn't `_client_id_` be the value of the client_id, instead of the literal string `_client_id`?

Reply:
No, this is correct. This is essentially a field in `$authorization_code_data`

Reply:
I see. I think in that case, we also need to check that when retrieving the `code`. Otherwise we might be returning the code of another oauth client, right?

Reply:
Right, but the codes are queried upon the basis of auth code externally, so only a single client for a single user should have it and surface in that case. So don't think that's needed in the usual sense of things?

Reply:
It's probably unlikely that there would be token clashes across different oauth clients, but I think we should guarantee that it never happens. I don't think this is related to this PR though, will open a ticket for that.